packages/@aws-cdk/aws-codebuild/lib/source.ts

```diff
@@ -793,6 +850,10 @@ class BitBucketSource extends ThirdPartyGitSource {
       sourceProperty: {
         ...superConfig.sourceProperty,
         location: this.httpsCloneUrl,
+        buildStatusConfig: {
+            context: this.buildStatusName,
+            targetUrl: this.buildStatusUrl,
```

Comment:
General comment: we use 2 spaces for indentation in the project, not 4.

This needs to be fixed everywhere in the PR (most notably, in the tests).